src/htmldiff.ts

```diff
@@ -95,6 +95,38 @@ function isEndOfAtomicTag(word: string, tag: string){
   return word.substring(word.length - tag.length - 2) === ('</' + tag);
 }
 
+const styleTagsRegExp = /^<(strong|em)/;
```

Comment:
Hmm, what about `h`, `i`, `b`, `q` tags, etc. There are more than just those too: `cite`, `blockquote`, `mark`, `dfn`, `sub`, `sup` are just some of the ones I can think of that might want similar behavior.